### ISSUE TITLE
Test for WCAG technique H36: Using alt attributes on images used as submit buttons

### ISSUE THREAD

**Issue by hannolans** [quailjs/quail#113]
Applies to content using image-based submit buttons. 

For input elements of type 'image', the alt attribute of the input element is used to provide a functional label. This label indicates the button's function, but does not attempt to describe the image. The label is especially important if there are multiple submit buttons on the page that each lead to different results.

The input element is used to create many kinds of form controls. Although the HTML and XHTML DTDs permits the alt attribute on all of these, it should be used only on image submit buttons. User agent support for this attribute on other types of form controls is not well defined, and other mechanisms are used to label these controls.
1. For all input elements that have a type attribute value of "image", check for the presence of an alt attribute.
2. Check that the alt attribute indicates the button's function.
